index.html

```diff
@@ -108,7 +108,7 @@ <h3>Implementation Examples</h3>
                                 <img src="https://docs.adyen.com/user/pages/docs/13.development-resources/github.svg" alt="" class="icon">
                                 <h5 class="title">Online Payments</h5>
                                 <div class="descrp">
-                                    <p>Performing an online payment using sessions</p>
+                                    <p>Perform an easy online payment using sessions</p>
```

Comment:
Looks good 👍🏾. 

I think 'easy' is relative. Maybe just say "Perform an online payment"

Reply:
Changed 👍 Thanks!